src/uharfbuzz/__init__.py

```diff
@@ -4,3 +4,95 @@
     __version__ = "0.0.0+unknown"
 
 from ._harfbuzz import *
+
+__all__ = [
+    "Blob",
+    "Buffer",
+    "BufferClusterLevel",
+    "BufferContentType",
+    "BufferFlags",
+    "Color",
+    "ColorLine",
+    "ColorStop",
+    "DrawFuncs",
+    "Face",
+    "Font",
+    "FontExtents",
+    "FontFuncs",
+    "GlyphExtents",
+    "GlyphFlags",
+    "GlyphInfo",
+    "GlyphPosition",
+    "HBObject",
+    "HarfBuzzError",
+    "Map",
+    "MapIter",
+    "OTColor",
+    "OTColorLayer",
+    "OTColorPalette",
+    "OTColorPaletteFlags",
+    "OTLayoutGlyphClass",
+    "OTMathConstant",
+    "OTMathGlyphPart",
+    "OTMathGlyphPartFlags",
+    "OTMathGlyphVariant",
+    "OTMathKern",
+    "OTMathKernEntry",
+    "OTMetricsTag",
+    "OTVarAxisFlags",
+    "OTVarAxisInfo",
+    "OTVarNamedInstance",
+    "PaintCompositeMode",
+    "PaintExtend",
+    "PaintFuncs",
+    "RepackerError",
+    "Set",
+    "SetIter",
+    "SubsetFlags",
+    "SubsetInput",
+    "SubsetInputSets",
+    "SubsetPlan",
+    "__version__",
+    "ot_color_glyph_get_layers",
+    "ot_color_glyph_get_png",
+    "ot_color_glyph_get_svg",
+    "ot_color_glyph_has_paint",
+    "ot_color_has_layers",
+    "ot_color_has_paint",
+    "ot_color_has_palettes",
+    "ot_color_has_png",
+    "ot_color_has_svg",
+    "ot_color_palette_color_get_name_id",
+    "ot_color_palette_get_colors",
+    "ot_color_palette_get_count",
+    "ot_color_palette_get_flags",
+    "ot_color_palette_get_name_id",
+    "ot_font_set_funcs",
+    "ot_layout_get_baseline",
+    "ot_layout_get_glyph_class",
+    "ot_layout_has_glyph_classes",
+    "ot_layout_has_positioning",
+    "ot_layout_has_substitution",
+    "ot_layout_language_get_feature_tags",
+    "ot_layout_lookup_get_glyph_alternates",
+    "ot_layout_script_get_language_tags",
+    "ot_layout_table_get_script_tags",
+    "ot_math_get_constant",
+    "ot_math_get_glyph_assembly",
+    "ot_math_get_glyph_italics_correction",
+    "ot_math_get_glyph_kerning",
+    "ot_math_get_glyph_kernings",
+    "ot_math_get_glyph_top_accent_attachment",
+    "ot_math_get_glyph_variants",
+    "ot_math_get_min_connector_overlap",
+    "ot_math_has_data",
+    "ot_math_is_glyph_extended_shape",
```

Comment:
All the `ot_*` functions above are deprecated, do we still need to include them here?

Reply:
I've just tested locally, and if we exclude the functions here they will be excluded from documentation, but would also be excluded from any existing star-imports (i.e. `from uharfbuzz import *`), and so it would be a breaking change.

Maybe this is okay if we release explicitly as breaking and there are alternatives though? (or even as v1.0.0, if the module enjoys any production use?)

Reply:
Alternatively there may be a way to make them show as deprecated in Sphinx

Reply:
(in some ways, even adding `__all__` at all is already a breaking change, but the only variables being excluded at the moment are re-exports from uharfbuzz's dependencies and the standard library, which I think we can consider a bug fix)